### PR TITLE
#86 HTTPExecutor enriched with the 'status' argument

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: src/mirakuru/http.py [hunk 1]
```diff
@@ -59,19 +65,22 @@ def __init__(self, command, url, **kwargs):
...
 
     def after_start_check(self):
-        """Check if defined url returns successful head."""
+        """Check if defined URL returns expected status to a HEAD request."""
         try:
             conn = HTTPConnection(self.host, self.port)
 
             conn.request('HEAD', self.url.path)
-            response = conn.getresponse()
+            status = str(conn.getresponse().status)
```
Comment (fizyk) [inline src/mirakuru/http.py]: on python3 it's a string (fully unicode enabled), on python2 it's a bytes

**Comment by fizyk** [comment]
:+1:

File: src/mirakuru/http.py (hunk 1 above)
Comment (pwilczynskiclearcode) [inline src/mirakuru/http.py]: [HTTPResponse.status](https://docs.python.org/3/library/http.client.html) seems to be an integer in both python versions.

Comment (fizyk) [inline src/mirakuru/http.py]: oh, I meant the str construct :)

**Pull request merged**